docs/source/installation/translations.rst

```diff
@@ -106,38 +113,6 @@ So if by any reason it doesn't work for you, feel free to :ref:`ask us <contact>
 .. _zh_Hant_TW: https://www.localeplanet.com/icu/zh-Hant-TW/index.html
 
 
-Contributing
```

Comment:
Except for the comment below, I've moved the rest of this section to the new page